gstcefsrc.cc

```diff
@@ -967,6 +967,8 @@ gst_cef_src_stop (GstBaseSrc *base_src)
 {
   GstCefSrc *src = GST_CEF_SRC (base_src);
 
+  GST_OBJECT_LOCK (src);
```

Comment:
Why is this needed? Taking the object lock then waiting is a bit awkward, it could mean  an for instance an application potentially blocking while trying to set a property from its main thread